tensorflow/tools/gcs_test/python/gcs_smoke.py

```diff
@@ -52,29 +52,75 @@ def create_examples(num_examples, input_mean):
 def create_dir_test():
   """Verifies file_io directory handling methods ."""
 
+  # Test directory creation
   starttime = int(round(time.time() * 1000))
   dir_name = "%s/tf_gcs_test_%s" % (FLAGS.gcs_bucket_url, starttime)
   print("Creating dir %s" % dir_name)
   file_io.create_dir(dir_name)
   elapsed = int(round(time.time() * 1000)) - starttime
   print("Created directory in: %d milliseconds" % elapsed)
+
   # Check that the directory exists.
   dir_exists = file_io.is_directory(dir_name)
+  assert dir_exists
   print("%s directory exists: %s" % (dir_name, dir_exists))
 
+  # Test recursive directory creation
+  starttime = int(round(time.time() * 1000))
+  recursive_dir_name = "%s/%s/%s" % (dir_name,
+                                     "nested_dir1",
+                                     "nested_dir2")
+  print("Creating recursive dir %s" % recursive_dir_name)
+  file_io.recursive_create_dir(recursive_dir_name)
+  elapsed = int(round(time.time() * 1000)) - starttime
+  print("Created directory recursively in: %d milliseconds" % elapsed)
+
+  # Check that the directory exists.
+  recursive_dir_exists = file_io.is_directory(recursive_dir_name)
+  assert recursive_dir_exists
+  print("%s directory exists: %s" % (recursive_dir_name, recursive_dir_exists))
+
   # List contents of just created directory.
```

Comment:
Update comment to "Create some contents in the just created directory and list the contents."